cmd/kubeadm/app/cmd/init.go

```diff
@@ -374,3 +387,37 @@ func createClientsetAndOptionallyWaitForReady(cfg *kubeadmapi.MasterConfiguratio
 	}
 	return client, nil
 }
+
+func getDirectoriesToUse(dryRun bool, defaultPkiDir string) (string, string, string, error) {
```

Comment:
I would like to find an alternative to a `bool` param but I don't have any great ideas (they're unintuitive).